src/main/java/io/supertokens/storage/mysql/Start.java

```diff
@@ -1210,9 +1237,18 @@ public void updateUserPhoneNumber_Transaction(TransactionConnection con, String
             throws StorageQueryException, UnknownUserIdException, DuplicatePhoneNumberException {
         Connection sqlCon = (Connection) con.getConnection();
         try {
-            PasswordlessQueries.updateUserPhoneNumber_Transaction(this, sqlCon, userId, phoneNumber);
-        } catch (SQLException e) {
-            // TODO: review these exceptions
+            int updated_rows = PasswordlessQueries.updateUserPhoneNumber_Transaction(this, sqlCon, userId, phoneNumber);
+
+            if (updated_rows != 1) {
+                throw new UnknownUserIdException();
+            }
+
+        } catch (UnknownUserIdException | SQLException e) {
+
+            if (e instanceof UnknownUserIdException) {
+                throw new UnknownUserIdException();
+            }
```

Comment:
You don't need to catch `UnknownUserIdException` and then throw the exact same thing again. Just don't catch `UnknownUserIdException` :)

Reply:
Agreed, I missed that.